ports/gemrb/port.json

```
@@ -19,10 +19,10 @@
         ],
         "image": {},
         "rtr": false,
-        "runtime": null,
+        "runtime": "pyxel_2.2.8_python_3.11.squashfs",
         "reqs": [],
```

Put the arch under requirements so that it doesnt show up on x86_64 devices, then remove it from the "arch" requirements:

```suggestion
        "reqs": ["aarch64"],
```

The problem is if you specify a "runtime" it then ignores the "arch". 😄

---

But I still keep the `"arch": [ "aarch64"]`?
https://github.com/PortsMaster/PortMaster-New/blob/2614c92ad4a460bf118ce80cdd968f01a4ece46d/ports/gemrb/port.json